tools/autograd/gen_python_functions.py

```diff
@@ -768,7 +768,7 @@ def get_py_formal_arg(arg):
         default = None
         if arg.get('default') is not None:
             default = arg['default']
-            if default == 'nullptr' or default == 'nullopt' or default == '{}':
+            if default == 'nullptr' or default == 'c10::nullopt' or default == '{}':
```

Comment:
I think I asked this before (but perhaps the comment went away because of the force push) -- why do you need the c10?

Reply:
Ah, responded on https://github.com/pytorch/pytorch/pull/15154 for this.

native_parse marks the default as c10::nullopt and then that gets propagated everywhere.

Rather than adding a bunch of `using` statements in generated code replacing the previous check since that isn't used for anything. Verified by tracing this line in code gen.

Reply:
I see; not a fan but you don't have to change it :).